pkg/trie/triedb/node.go

```diff
@@ -252,7 +252,7 @@ type ChildReference interface {
 type (
 	// HashChildReference is a reference to a child node that is not inlined
 	HashChildReference struct {
-		hash common.Hash
+		Hash common.Hash
```

Comment:
maybe just make `HashChildReference` a `common.Hash` at this point?

Reply:
Done! and I'll do the same with similar types in `codec/node.go` in a different PR